kernel/mlx5/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+authors = ["Ramla Ijaz <ijazramla@gmail.com>"]
+name = "mlx5"
+description = "Support for the Mellanox connectx-5 NIC and driver"
```

Comment:
nit: is this crate not the driver itself? Your comment makes it sound like this is a support crate and the driver might be elsewhere